android/source/VideoLocker/src/org/edx/mobile/module/db/impl/DbHelper.java

```diff
@@ -53,8 +56,11 @@ public void onCreate(SQLiteDatabase db) {
     @Override
     public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
         try{
-            String upgradeQuery = "ALTER TABLE "    + DbStructure.Table.DOWNLOADS 
-                    + " ADD COLUMN " + DbStructure.Column.UNIT_URL + " TEXT ";
+            String upgradeQuery = "ALTER TABLE "    + DbStructure.Table.DOWNLOADS
+                    + " ADD COLUMN " + DbStructure.Column.UNIT_URL + " TEXT "
+                    + " ADD COLUMN " + DbStructure.Column.URL_HIGH_QUALITY + " TEXT "
+                    + " ADD COLUMN " + DbStructure.Column.URL_LOW_QUALITY + " TEXT "
```

Comment:
This might not work as the Db version is changed to 3 and the check for old version is 2.